deluge/common.py

```diff
@@ -56,6 +58,50 @@
 
 log = logging.getLogger(__name__)
 
+# A TLS ciphers list.
```

Comment:
Perhaps instead of adding more code to common.py create a new crypto.py or crypto_utils.py file

Reply:
Done. Moved to `crypto_utils.py`